d2l-table-wrapper.html

```diff
@@ -163,6 +165,7 @@
 				Polymer.RenderStatus.afterNextRender(this, function() {
 					this._registerTableObserver();
 					this._applyClassNames();
+					this._applySubheadingTop();
```

Comment:
Suggestion: do this as a [property observer](https://www.polymer-project.org/1.0/docs/devguide/observers) on the `sticky-headers` property. That way if the developer using `d2l-table` adds or removes the `sticky-headers` attribute after the table has already rendered, your code will run. Observers should always get called on initialization anyway, so it shouldn't change how you have things working now.